### DISCUSSION TITLE
2.4ghz is 3 to 4 times faster than 5ghz

### DISCUSSION THREAD

**Discussion by yoshideryu** [RaspAP/raspap-webgui#946] [Q&A]
2.4ghz is 3 to 4 times faster than 5ghz could this be a hardware issue my pi has a powered hub using 2 TP-Link t2u plus. on 2.4ghz my speed is around 40mbps on a test 5ghz is around 12mpbs max let me know if you need more info.

**Comment by billz** [comment]
Depends on several factors including adapter hardware, driver, testing method, etc. For comparison, 5GHz ac with the RPi4's onboard chipset yielded speeds in the [75-100Mb/s range](https://github.com/RaspAP/raspap-webgui/issues/450#issuecomment-568931205). 

TP-link adapters are generally not recommended for [reasons described here](https://github.com/morrownr/USB-WiFi). They are often not worth the trouble.

**Reply by yoshideryu** [reply]
what would you recommend its a pi 3b   I had a issue with a lot of realtech  drivers not being able to run 2 of the same dongle at the same time  the 8822bu will not run in a pair let me know it would make my day

**Reply by billz** [reply]
The link cited above is your best resource for the Pi. See also [RaspAP tools](https://github.com/RaspAP/raspap-tools) if you need to install something other than an in-kernel driver.
